### PR TITLE
PLATFORM-3462: remove old sitemaps code

### PR DESCRIPTION
**Pull request by themech**

### CODE REVIEW

File: extensions/wikia/SitemapXml/SpecialSitemap_body.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		global $wgOut;
+
+		// keeping the checks for old sitemaps for now so we return 404s
+		$showIndex = strpos( $subpage, '-index.xml' ) !== false;
+
+		$forceOldSitemap = strpos( $subpage, '-oldsitemapxml-' ) !== false;
+		$forceNewSitemap = strpos( $subpage, '-newsitemapxml-' ) !== false;
+
+		if ( ( $showIndex && !$forceOldSitemap ) || $forceNewSitemap ) {
+			$wgOut->disable();
```
Comment (Grunny) [inline extensions/wikia/SitemapXml/SpecialSitemap_body.php]: We can avoid the global with `$this->getOutput()->disable();`

File: extensions/wikia/SitemapXml/SpecialSitemap_body.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+	/**
+	 * show 404
+	 *
+	 * @access private
+	 */
+	private function print404() {
+		global $wgOut;
+
+		$wgOut->disable();
```
Comment (Grunny) [inline extensions/wikia/SitemapXml/SpecialSitemap_body.php]: Same here.

**Review by Grunny — APPROVED** [review]
Minor comment, otherwise looks good.

File: extensions/wikia/SitemapXml/SpecialSitemap_body.php (hunk 1 above)
Comment (themech) [reply extensions/wikia/SitemapXml/SpecialSitemap_body.php]: Done. In my defence, I just moved this file :)

File: extensions/wikia/SitemapXml/SpecialSitemap_body.php (hunk 2 above)
Comment (themech) [reply extensions/wikia/SitemapXml/SpecialSitemap_body.php]: ditto

**Pull request merged**